code/controllers/subsystem/jobs.dm

```diff
@@ -351,6 +351,7 @@ var/datum/subsystem/job/SSjob
 			player.ready = 0
 			unassigned -= player
 			ticker.mode.antag_candidates -= player.mind
+			alert(player, "You were returned to the lobby because your preference unavailable.","You were returned to the lobby","OK")
```

Comment:
http://www.byond.com/docs/ref/#/proc/alert
This sleeps the current proc until the user clicks one of the named buttons. As with input(), the first argument may be entirely left out.